ui/src/org/theGraphApi.ts

```diff
@@ -61,6 +63,74 @@ export interface Org {
   projectCount?: number;
 }
 
+export interface Safe {
+  id: string;
+  owners: string[];
+  threshold: number;
+}
+
+export async function getSafesByOwner(owners: string[]): Promise<Safe[]> {
```

Comment:
This function is only ever called with one owner as an argument. Maybe we should convert it to
```suggestion
export async function getSafesByOwner(owner: string): Promise<Safe[]> {
```

Reply:
8221747c53fdcbbeb5f3c0743bc27a8f8442b02d